SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerMultiPossessionTest.cpp

```diff
@@ -40,6 +40,12 @@ ACrossServerMultiPossessionTest::ACrossServerMultiPossessionTest()
 	Description = TEXT("Test Cross-Server Multi Controllers Possess 1 Pawn");
 }
 
+void ACrossServerMultiPossessionTest::CreateCustomContentForMap()
+{
+	Super::CreateCustomContentForMap();
```

Comment:
I assume we are relying on the base class to have the exact same settings we need here?

Reply:
Thanks, as discussed elsewhere I'm not 100% keen on this approach, I'd prefer each test to clearly set its own parameters. 

However I think you will be addressing this in the other issue?

Reply:
Not sure what issue you're referring to? But what you're saying does make sense as I guess the settings are one of the first things you check when reading a test, I've moved the settings out from the base class to the derived tests.